### PR TITLE
Vault Program

### PR DESCRIPTION
**Pull request by uklok-agent**
## Description

The vault program was included. It's a simple contract that allows you to store and withdraw any token.

Note: This is not production-ready, just an initial version to implement bounties.

## Additional Information

- [x] Unit testing for all programs fully working.
- [x] UI for Vault program implemented.
- [x] Initial `PlayButton` for entrance game.

---
Your SNS/address: [uklok.sol](https://www.sns.id/domain?domain=uklok)

### CODE REVIEW

**Review by jiro141 — COMMENTED** [review]
.

File: web/public/img/push-me/play.svg
Comment (jiro141) [inline web/public/img/push-me/play.svg:1]: una pregunta, tu crees que es mejor usar la imagen como svg cierto?

Comment (uklok-agent) [reply web/public/img/push-me/play.svg:1]: Yes, all these images should be `svg`.

**Review by jiro141 — COMMENTED** [review]
listo

**Review by jiro141 — APPROVED** [review]

**Pull request merged**